### PR TITLE
Website rework

### PR DESCRIPTION
**Pull request by lumi2021**
# The Silk.NET website remade from scratch

As requested by Perksey, it is now built on Docusaurus. It style is completely made from scratch based on the current one.

To run the project:
- Make sure Node.js and NPM are installed and up to date;
- Make sure you have the last version of the fork;
- Run the nuke command with "website"

Unfortunately, Docusaurus uses git history information to set doc page's author, so I'm the author of everything now lol. It can be solved only if the respective authors do some changes on their `.md` files.

Any issues, questions or anything, pls comment here.

<sub>_Perksey-senpai pls accept my PR_ 👉👈😖</sub>

### CODE REVIEW

**Comment by Perksey** [comment]
> Unfortunately, Docusaurus uses git history information to set doc page's author, so I'm the author of everything now lol. It can be solved only if the respective authors do some changes on their .md files.

I'll try and sort this out if accepted.

**Comment by Perksey** [comment]
Is this ready for review?

File: build/nuke/Build.Website.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+using Nuke.Common.Tools.Npm;
+
+partial class Build
+{
+    static string workingDirectory = "./website";
+
+    Target Website => _ => _
+        .Executes(() =>
+        {
+            Console.WriteLine("Verifying dependences:\n");
```
Comment (GreemDev) [inline build/nuke/Build.Website.cs:19]: Dependenc**i**es

Comment (GreemDev) [reply build/nuke/Build.Website.cs:19]: And also the related method

**Comment by lumi2021** [comment]
> Is this ready for review?
@Perksey 

uhhh... yeah? what is the problem?

**Comment by Perksey** [comment]
I'm closing this PR. Too many of the review comments on the last PR have not been addressed such that this would take too much time to review, and too much time to fix ourselves. I think this work item has exhausted the amount of time the team is able to dedicate towards it. We may pick this back up in the future.

**Pull request closed without merging**